install.sh

```diff
@@ -24,6 +24,9 @@ do
     fi
 done
 
+# Make profile.d dir first
+mkdir -p /etc/profile.d
```

Comment:
I'm not sure just creating the profile.d folder is enough. On Fedora, my /etc/profile file has some code that runs .sh files in profile.d. On Liz's Mac, that code is not there.

Would it make sense to instead check if the /etc/profile.d folder exists (-d) and add docker-uid.sh script if it does, otherwise append the export to /etc/profile? I think that'll support Linux distros that don't have a /etc/profile.d as well.